coax/proba_dists/_normal.py

```diff
@@ -81,17 +100,24 @@ def check_shape(x, name, flatten):
                 raise ValueError(f"expected {name}.shape: (?, {expected}), got: {x.shape}")
             if flatten:
                 x = x.reshape(x.shape[0], -1)  # batch-flatten
+            if name.startswith("logvar"):
+                x = jnp.clip(x, *self.clip_logvar)
             return x
 
         def sample(dist_params, rng):
             mu = check_shape(dist_params['mu'], name='mu', flatten=True)
             logvar = check_shape(dist_params['logvar'], name='logvar', flatten=True)
 
             X = mu + jnp.exp(logvar / 2) * jax.random.normal(rng, mu.shape)
+            if self.squash:
+                X = jnp.tanh(X) * self._scale + self._offset
```

Comment:
I think used an env wrapper for this in the past. Have you seen [coax.wrappers.BoxActionsToReals](https://coax.readthedocs.io/en/latest/coax/wrappers.html#coax.wrappers.BoxActionsToReals)?